index.html

```diff
@@ -2999,6 +3000,20 @@
         <div id="eventBuffs" alt="eventBuffs" label="eventBuffs" style="color: lime;">Buffs: </div>
         <a href="#" target="_blank" id="happyHolidays" alt="happyHolidays" label="happyHolidays" style="color: greenyellow"></a>
     </div>
+    <div class="subtabContent subtabDisplayFlex" id="languagesubtab" alt="languagesubtab" label="languagesubtab">
+        <div class="languagesMenu">
+            <div id="languageOptions" class="languageOptions">
+                <!-- Populated by i18n.ts, `buildLanguageTab` -->
+            </div>
+        </div>
+        <div class="languageContribution">
+            <p>
+                Interested in providing a translation for your language?  Visit the <a href="https://www.discord.gg/ameCknq">#translations channel on our Discord!</a>
+            </p>
+            <p>Visit our Crowdin project at <a href="https://crowdin.com/project/synergism-official">https://crowdin.com/project/synergism-official</a>!</p>
+            <p>Contribute directly through Github at <a href="https://github.com/Pseudo-Corp/SynergismOfficial">https://github.com/Pseudo-Corp/SynergismOfficial</a>!</p>
```

Comment:
This text should also be translate-able. I guess having a giant method in i18n.ts that changes text in the html file on load/language change would suffice. You can add text to translations/default.json and I will upload them to crowdin.

Reply:
That's totally fair.

Before I go through the trouble, is this text about what you'd hope to have for the tab?  Or would different info be preferred?

Reply:
What's the current planned strategy for doing this with other index.html strings, if there is one?  I probably shouldn't diverge too much for whatever that strategy may be.

Even if not everything is localized, I imagine the tab titles and subtab titles should probably be localized... and handling for that would face very similar issues.

My _first_ thought is using element IDs as i18n string keys, since they're usually unique on the page, but if you have something else in mind, feel free to suggest something more aligned to your expectations.

Reply:
Crowdin should be replaced with poeditor (https://poeditor.com/join/project/1up4tT24vG) now, but everything else seems fine.

Reply:
I could see element keys be named (or have an i18n- attribute) so we don't have to move all the text out of the html file.

```html
<div i18n="key.something.name" i18n-replace="{{name:abc}}" />
```

where the key would be
```json
{
  "key": {
    "something": {
      "name": "Your name is {{name}}"
    }
  }
}
```

although better left off to another PR